lib/src/Uncertainty/Process/ARMAFactoryImplementation.cxx

```diff
@@ -155,24 +155,12 @@ ARMA ARMAFactoryImplementation::build(const TimeSeries & timeSeries) const
   throw NotYetImplementedException(HERE) << "In ARMAFactoryImplementation::build(const TimeSeries & timeSeries) const";
 }
 
-ARMA ARMAFactoryImplementation::build(const TimeSeries & timeSeries,
-                                      Point & informationCriteria) const
-{
-  throw NotYetImplementedException(HERE) << "In ARMAFactoryImplementation::build(const TimeSeries & timeSeries, Point & informationCriteria) const";
-}
-
 /* Build method */
 ARMA ARMAFactoryImplementation::build(const ProcessSample & sample) const
 {
```

Comment:
I thought returning a Point for the criteria is specific to WhittleFactory criteria, so I ditched them. Anyways it's not available from ARMAFactory interface. But as there are no other ARMAFactoryImplementation other than WhittleFactory it's hard to tell.